src/onepasswordconnectsdk/connect.py

```diff
@@ -0,0 +1,39 @@
+class PathBuilder:
+    def __init__(self, version: str = "/v1"):
+        self.path: str = version
+
+    def build(self) -> str:
+        return self.path
+
+    def vaults(self, uuid: str = None) -> 'PathBuilder':
+        self._append_path("vaults")
+        if uuid is not None:
+            self._append_path(uuid)
+        return self
+
+    def items(self, uuid: str = None) -> 'PathBuilder':
+        self._append_path("items")
+        if uuid is not None:
+            self._append_path(uuid)
+        return self
+
+    def files(self, uuid: str = None) -> 'PathBuilder':
+        self._append_path("files")
+        if uuid is not None:
+            self._append_path(uuid)
+        return self
+
+    def content(self) -> 'PathBuilder':
```

Comment:
`[non-blocking]`
Is it worth writing a test for this function as well? I see that this is the only one that doesn't have tests. 🤔

Reply:
added additional test